pkg/nodeps/values.go

```diff
@@ -23,6 +23,11 @@ const (
 	Postgres = "postgres"
 )
 
+// MySQLRemoveDeprecatedMessage is used to remove the deprecation from stderr when using mysql tools with MariaDB 11.x and later.
+// mysql: Deprecated program name. It will be removed in a future release, use '/usr/bin/mariadb' instead
+// mysqldump: Deprecated program name. It will be removed in a future release, use '/usr/bin/mariadb-dump' instead
+const MySQLRemoveDeprecatedMessage = " 2> >(grep -v 'Deprecated program name.' >&2) "
```

Comment:
Very fancy. For all my years of bash and shell I've never seen this notation before! I had to ask chatgpt to explain it to me.

Reply:
I found it here https://stackoverflow.com/a/52575087/8097891